### PR TITLE
14 Link the database to the about.json route

### LINKED ISSUE #14: Link the database to the about.json
**Short description:**
You need to call the database to list the available services.

**Describe the solution you want**.
Use prisma and make a call to list the available services and each of their actions/reactions.

### PR DESCRIPTION
**Pull request by ZQUEMA**
# Purpose

The purpose of this pull request is to link the database to the about.json route.

Fixes #14 

## Added features

* link the database to the about.json route and display each services and theirs actions/reaction

### CODE REVIEW

File: server/server_app.js [hunk 1]
```diff
@@ -52,14 +52,37 @@ app.get('/', (req, res) => {
...
+            select: {
+              name: true,
+              description: true,
+              isEnable: true
+            }
+          }
+        }
+      })
+    )
     // TODO fetch the services data from DB with the IP of the client
```
Comment (LucasTesnier) [inline server/server_app.js]: Please remove it.

**Review by LucasTesnier — CHANGES_REQUESTED** [review]
Please remove useless comment...

**Review by GuyomT — DISMISSED** [review]
LGTM

**Review by tbellicha — APPROVED** [review]

**Review by LucasTesnier — APPROVED** [review]
AMaZiNG wOrK !§!

**Pull request merged**